src/apify_client/clients/resource_clients/webhook_collection.py

```diff
@@ -0,0 +1,70 @@
+from typing import Any, Dict, List, Optional
+
+from ..base.resource_collection_client import ResourceCollectionClient
+from .webhook import prepare_webhook_representation
+
+
+class WebhookCollectionClient(ResourceCollectionClient):
+    """Sub-client for manipulating webhooks."""
+
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        """Initialize the WebhookCollectionClient."""
+        super().__init__(*args, resource_path='webhooks', **kwargs)
+
+    def list(self, *, limit: Optional[int] = None, offset: Optional[int] = None, desc: Optional[bool] = None) -> Dict:
+        """List the available webhooks.
+
+        https://docs.apify.com/api/v2#/reference/webhooks/webhook-collection/get-list-of-webhooks
+
+        Args:
+            limit (int, optional): How many webhooks to retrieve
+            offset (int, optional): What webhook to include as first when retrieving the list
+            desc (bool, optional): Whether to sort the webhooks in descending order based on their date of creation
```

Comment:
btw, these types in args are cool. I wasn't doing it before but I will in the future. Thx